src/components/mobile-menu/mobile-menu.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Link } from "gatsby";
+
+import { NavMenu } from "../nav-menu";
+import { Button } from "../button";
+import { MenuMobileIcon } from "../icons/menu-icon";
+import { CloseIcon } from "../icons/close-icon";
+import { LinkedInIcon } from "../icons/linked-in/";
+import { InstagramIcon } from "../icons/instagram";
+import { TwitterIcon } from "../icons/twitter";
+import { GithubIcon } from "../icons/github";
+import { classnames } from "../../helpers";
+
+import * as styles from "./mobile-menu.module.scss";
+
+const MobileMenu = ({ onClick, status }) => (
+  <>
+    <div className={classnames(styles.container, { [styles.menuOpen]: status })} >
+      <div className={classnames(styles.buttonIconPosition, { [styles.openButton]: status })}>
+        <Button onClick={onClick} className={styles.buttonIconStyles} >
+        <MenuMobileIcon />
+        </Button>
+      </div>
+      {status && 
```

Comment:
What is the meaning of the status?

Reply:
I changed it to openOrClose, the status shows the menu when the user click on the button

Reply:
I think this name is a bit confusing, if when the variable is true you show the menu then the variable should be `openMenu` or `showMenu`, if when the variable is true you don't show the menu then the variable should be 'closeMenu'

Reply:
Done